build.sh

```diff
@@ -23,7 +23,7 @@ sudo apt-get install -y \
 
 sudo mkdir /usr/local/php7
 
-git clone https://github.com/php/php-src.git
+git clone -b PHP-7.0.0 --depth=1 https://github.com/php/php-src.git
 cd php-src
 git checkout PHP-7.0.0
```

Comment:
what's the difference (I did this before because only afterwards I realized version 7.0.2), but @kasparsd here you are:

```
root@625f399b0f83:~# ls -la
total 16
drwx------   1 root root 4096 Jan  8 15:52 .
drwxr-xr-x   1 root root 4096 Jan  8 15:46 ..
-rw-r--r-- 498 root root  570 Jan 31  2010 .bashrc
-rw-r--r-- 498 root root  140 Nov 19  2007 .profile
root@625f399b0f83:~# git clone -b PHP-7.0.1 --depth=1 https://github.com/php/php-src.git
Cloning into 'php-src'...
remote: Counting objects: 18070, done.
remote: Compressing objects: 100% (15074/15074), done.
remote: Total 18070 (delta 3610), reused 10633 (delta 2842), pack-reused 0
Receiving objects: 100% (18070/18070), 19.47 MiB | 834.00 KiB/s, done.
Resolving deltas: 100% (3610/3610), done.
Checking connectivity... done.
root@625f399b0f83:~# cd php-src/
root@625f399b0f83:~/php-src# git status
On branch PHP-7.0.1
Your branch is up-to-date with 'origin/PHP-7.0.1'.
nothing to commit, working directory clean
root@625f399b0f83:~/php-src# git branch -vla
* PHP-7.0.1                4054ec6 Refactor password_hash to use random_bytes internally to generate salts
  remotes/origin/PHP-7.0.1 4054ec6 Refactor password_hash to use random_bytes internally to generate salts
root@625f399b0f83:~/php-src# git checkout PHP-7.0.1
Already on 'PHP-7.0.1'
Your branch is up-to-date with 'origin/PHP-7.0.1'.
```